network-api/networkapi/mozfest/migrations/0017_update_wide_images.py

```diff
@@ -0,0 +1,65 @@
+# # Generated by Django 3.0.14 on 2021-08-10 18:19
+
+from django.db import migrations
+import wagtail.core.blocks
+import wagtail.core.blocks.static_block
+import wagtail.core.fields
+import wagtail.images.blocks
+import wagtailmedia.blocks
+
+
+from networkapi.mozfest.models import MozfestPrimaryPage
+
+
+
+def update_annotated_image_blocks(qs):
+    # Loop through all pages
+    for page in qs:
+        print("\t", page)
+        # If this page doesn't have a Image that needs update, don't save it.
+        # This will speed up how fast this task can run.
+        needs_saving = False
+        # Only loop through streamfield data if there's a `body` field on the page
+        if page.body:
+            # Looping through raw_data is a List of blocks.
+            for block in page.body.raw_data:
```

Comment:
Note that this only works "for now". If some future version of Wagtail changes the property name, this migration will error out. Let's at the very least test for `hasattr(page.body, 'raw_data')` before we loop through it.

Reply:
Thank you both for taking the time to look at this PR and for the catch. I have added the check and am marking this conversation as resolved!